docs/release-notes/v1.1.rst

```diff
@@ -1,6 +1,178 @@
 .. _gammapy_1p1_release:
 
-1.0 (unreleased)
-----------------
+1.1 (May XXth 2023)
+-------------------
 
-- No changes yet
+- Released May XXth, 2023
+- 14 contributors
+- 69 pull requests since v1.0.1 (not all listed below)
+- 85 closed issues
+
+Summary
+~~~~~~~
+
+This release introduces a number of new features as well as some performance improvements.
+
+Support for energy dependent temporal models is introduced. Only template models are supported.
+They use a `RegionNDMap` object with an energy and a time axis. Note that these models
+are meant for event simulation cannot be used for modeling and fitting
+
+A tutorial demonstrating how to simulate events using this type of models has been added.
+
+Support for multiprocessing is improved. `FluxPointsEstimator` and `LightCurveEstimator`
+can now run on several cores. This is defined with the `n_jobs` parameter that can be set on
+init. The backend used by default is the python multiprocessing. Support for the ray
+multiprocessing is proposed as a prototype as well. This feature requires installing
+the `ray` library and can be defined with tjhe `parallel_backend` property.
+This interface is also used by the `DatasetsMaker` that performs the data reduction
+loop.
+
+A first step in the separation of the internal data model and the GADF format is introduced
+in this release in the handling of the pointing. This is part of a larger that will be
+implemented in the coming feature releases to allow I/O with multiple data formats.
+
+New features
+~~~~~~~~~~~~
+
+*gammapy.data*
+
+- A new function `~gammapy.data.get_irfs_features()` can extract the features of IRFs (such as energy
+  resolution bias or PSF radius) of a list of `Observation`. The output list can then be passed
+  to the function `~gammapy.utils.cluster.hierarchical_clustering()` which will find clusters
+  of IRF features allowing to combine `Observation` with similar IRF characteristics during
+  analysis.
+
+*gammapy.maps*
+
+- A `Map.reorder_axes` helper method has been introduced.
+- Dot product is now supported with the `Map.dot()` function. It applies the dot product on
+  the axes sharing the same name. It can also be called via the `@` operator.
+- A `WcsNDMap.to_region_nd_map_histogram()` helper method is introduced to compute
+  the histogram of the map values along the spatial dimensions for each non spatial axes
+  bins.
+
+*gammapy.modeling*
+
```

Comment:
Mention of energy dependent temporal model feature?

Reply:
yes I have to add this